@hubblecommerce/hubble/core/components/swComponents/slots/buy-box-slot.vue

```diff
@@ -16,6 +16,9 @@ export default {
         product() {
             return this.content && this.content.data && this.content.data.product && mappingProduct({ product: this.content.data.product });
         },
+        options() {
+            return this.content && this.content.data && this.content.data.configuratorSettings;
+        }
```

Comment:
Do not create an extra property for options. Instead set the configuratorSettings to the mappingProduct function above.

Reply:
Fixed in [27f554c](https://github.com/hubblecommerce/hubble-frontend-pwa/commit/27f554c1061916e8fef173ff88e0efc1db04ef0e)